tensorboard/backend/application.py

```diff
@@ -264,6 +264,14 @@ def __init__(self, plugins, path_prefix=''):
           path = (self._path_prefix + DATA_PREFIX + PLUGIN_PREFIX + '/' +
                   plugin.plugin_name + route)
         self.data_applications[path] = app
+    
+    # Wildcard routes will be checked in the given order, so we sort them
+    # longest to shortest so that a more specific route will take precedence
+    # over a more general one (e.g., a catchall route `/*` should come last).
```

Comment:
This seems important; want to test it?

Reply:
+1, added a test--thanks for the reminder!